config/routes.rb

```diff
@@ -38,5 +38,10 @@
     post 'office_social_workers', on: :collection
   end
 
+  get 'verify' => 'verifications#index', as: :verify
+  post 'send_code' => 'verifications#send_code', as: :send_code
+  post 'check_code' => 'verifications#check_code', as: :check_code
```

Comment:
Thoughts on this? Pseudocode, I didn't try it. :smile: 

```ruby
resource :verifications, only: [] do
  on :collection do
   get :verify, action: :index
   post :send_code
   post :check_code
  end
end
```

The route helper names would change/be auto-generated.

Reply:
I thought of something like that, although this is a very weak idea of a "resource"... it doesn't have any of the standard CRUD methods or map in any ORM sense. The verifications controller here feels more like a container for some relevant custom methods. It's functionally equivalent though and I don't have a strong preference.

Reply:
Cool, your reasoning makes sense to me. :+1: